repository/inmemory/routing.go

```diff
@@ -22,12 +24,15 @@ func NewRoutingRepository() repository.RoutingRepository {
 	return &routingRepository{}
 }
 
-func (r *routingRepository) Add(jobCategory string, queueName string) error {
+func (r *routingRepository) Add(jobCategory string, queueName string) (bool, error) {
 	rs.Lock()
 	defer rs.Unlock()
 
-	rs.m[jobCategory] = queueName
-	return nil
+	if rs.m[jobCategory] != queueName {
+		rs.m[jobCategory] = queueName
+		return true, nil
+	}
```

Comment:
[QUESTION] Should we call `updateRevision` here? It is the same for `DeleteByJobCategory`.

Reply:
Right!  And the reason why the test was passing is a typo [here](https://github.com/fireworq/fireworq/pull/192/commits/b76fedd2732849ac4571b276b6ff22de73287af6#diff-c3b81b4ef722275c696aa684cacaddf9b115a7456ebe95cc58e080b475ba062bL259).  Fixed in [b76fedd](https://github.com/fireworq/fireworq/pull/192/commits/b76fedd2732849ac4571b276b6ff22de73287af6).